misc/windows-deploy/hostsetup.ps1

```diff
@@ -24,5 +24,8 @@ $containerGateway = (Get-NetNat).InternalIPInterfaceAddressPrefix | %{ $_ -repla
 
 netsh interface portproxy add v4tov4 listenaddress=$containerGateway listenport=51679 connectaddress=127.0.0.1 connectport=51679 protocol=tcp
 
-route -p delete 169.254.170.2/32
+$existingRoute=route print 169.254.170.2 -4 | Sort-Object -Property "IPv4 Route Table"
+if ($existingRoute -Match "169.254.170.2") {
+   route -p delete 169.254.170.2/32
```

Comment:
Seems like this is not required for the `delete` command. From [docs](https://technet.microsoft.com/en-us/library/bb490991.aspx):

> When used with the add command, the specified route is added to the registry and is used to initialize the IP routing table whenever the TCP/IP protocol is started. By default, added routes are not preserved when the TCP/IP protocol is started. When used with the print command, the list of persistent routes is displayed. This parameter is **ignored** for all other commands.